### PR TITLE
IBX-7503: Fixed custom attributes

### PR DESCRIPTION
**Pull request by dew326**
| Question           | Answer
| ------------------ | ------------------
| **JIRA issue**     | https://issues.ibexa.co/browse/IBX-7503
| **Bug/Improvement**| yes
| **New feature**    | no
| **Target version** | 4.5
| **BC breaks**      | no
| **Tests pass**     | yes
| **Doc needed**     | no

**TODO**:
- [x] Implement feature / fix a bug.
- [x] Implement tests.
- [x] Fix new code according to Coding Standards (`$ composer fix-cs`).
- [x] Ask for Code Review.

### CODE REVIEW

File: src/bundle/Resources/public/js/CKEditor/custom-attributes/custom-attributes-editing.js [hunk 1]
```diff
@@ -4,6 +4,12 @@ import Widget from '@ckeditor/ckeditor5-widget/src/widget';
 import IbexaCustomAttributesCommand from './custom-attributes-command';
 import { getCustomAttributesConfig, getCustomClassesConfig } from './helpers/config-helper';
 
+const configElementsMapping = {
```
Comment (Gengar-i) [inline src/bundle/Resources/public/js/CKEditor/custom-attributes/custom-attributes-editing.js:7]:
```suggestion
export const configElementsMapping = {
```

File: src/bundle/Resources/public/js/CKEditor/custom-attributes/custom-attributes-editing.js [hunk 2]
```diff
@@ -194,3 +202,4 @@ class IbexaCustomAttributesEditing extends Plugin {
 }
 
 export default IbexaCustomAttributesEditing;
+export { configElementsMapping };
```
Comment (Gengar-i) [inline src/bundle/Resources/public/js/CKEditor/custom-attributes/custom-attributes-editing.js:205]:
```suggestion
```

File: src/bundle/Resources/public/js/CKEditor/custom-attributes/custom-attributes-editing.js (hunk 1 above)
Comment (Gengar-i) [reply src/bundle/Resources/public/js/CKEditor/custom-attributes/custom-attributes-editing.js:7]: What do you think?

File: src/bundle/Resources/public/js/CKEditor/custom-attributes/custom-attributes-editing.js [hunk 3]
```diff
@@ -194,3 +202,4 @@ class IbexaCustomAttributesEditing extends Plugin {
 }
 
 export default IbexaCustomAttributesEditing;
```
Comment (Gengar-i) [inline src/bundle/Resources/public/js/CKEditor/custom-attributes/custom-attributes-editing.js:204]:
```suggestion
export { default as IbexaCustomAttributesEditing, configElementsMapping };
```

Comment (Gengar-i) [reply src/bundle/Resources/public/js/CKEditor/custom-attributes/custom-attributes-editing.js:204]: Or like this

**Review by Gengar-i — CHANGES_REQUESTED** [review]

File: src/bundle/Resources/public/js/CKEditor/custom-attributes/helpers/config-helper.js [hunk 4]
```diff
@@ -1,5 +1,11 @@
+import { configElementsMapping } from '../custom-attributes-editing';
+
 const headingsList = ['heading1', 'heading2', 'heading3', 'heading4', 'heading5', 'heading6'];
+const findConfigName = (elementName) => {
+    const configName = Object.entries(configElementsMapping).find(([, value]) => value === elementName);
 
+    return configName ? configName[0] : elementName;
```
Comment (GrabowskiM) [inline src/bundle/Resources/public/js/CKEditor/custom-attributes/helpers/config-helper.js:7]:
```suggestion
    return configName?.[0] ?? elementName;
```
?

File: src/bundle/Resources/public/js/CKEditor/plugins/elements-path.js [hunk 5]
```diff
@@ -71,7 +94,15 @@ class IbexaElementsPath extends Plugin {
...
             'click',
             () => {
-                this.editor.model.change((writer) => writer.setSelection(element, 'in'));
+                let placement = 'in';
+
+                this.isTableCellSelected = element.name === 'tableCell';
+
+                if (this.isTableCellSelected) {
+                    placement = 'on';
+                }
```
Comment (GrabowskiM) [inline src/bundle/Resources/public/js/CKEditor/plugins/elements-path.js:103]:
```suggestion
                this.isTableCellSelected = element.name === 'tableCell';
                
                const placement = this.isTableCellSelected ? 'on' : 'in';
```

**Review by GrabowskiM — CHANGES_REQUESTED** [review]

**Review by lucasOsti — APPROVED** [review]

**Review by GrabowskiM — APPROVED** [review]

**Review by Gengar-i — APPROVED** [review]

**Pull request merged**